### PR TITLE
[ci]  Fixes #2 Added file travis and Makefile

### LINKED ISSUE #2: Setup Travis builds

### PR DESCRIPTION
**Pull request by m4yk3ldev**

### CODE REVIEW

File: src/Lykke.Service.Skycoin.Sign/Startup.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 options.Logs = logs =>
                 {
                     logs.AzureTableName = "Skycoin.SignLog";
-                    logs.AzureTableConnectionStringResolver = settings => settings.Skycoin.SignService.Db.LogsConnString;
+                    logs.AzureTableConnectionStringResolver = settings => settings.SkycoinSignService.Db.LogsConnString;
```
Comment (olemis) [inline src/Lykke.Service.Skycoin.Sign/Startup.cs]: Revert this change and update namespaces accordingly.

**Review by olemis — CHANGES_REQUESTED** [review]

File: src/Lykke.Service.Skycoin.Sign/Startup.cs (hunk 1 above)
Comment (m4yk3ldev) [reply src/Lykke.Service.Skycoin.Sign/Startup.cs]: @olemis  the names of functions can not have a point, they are limited.

https://github.com/simelotech/exchange_sign/pull/3/commits/4a7e89ec58927a0ccdb63f2109b2ccbf705b23ab#diff-1c50f3c7059db3cf6563050da7caac36
